sr_ur_launch/config/configuration.yaml

```diff
@@ -0,0 +1,39 @@
+robot_hardware:
+  - ra_sr_ur_robot_hw
+
+ra_sr_ur_robot_hw:
```

Comment:
This is duplicating the information in right_ur_arm_controller.yaml. Maybe that file should be removed?